utils/bin/diagrammer

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# This may be a brittle way to find $(root_dir)/utils/bin, is there a better way?
+path=`dirname "$0"`
+cmd="java -cp ${path}/diagrammer.jar dotvisualizer.FirrtlDiagrammer ${@:1}"
+eval $cmd
```

Comment:
I think there is a way to stick a shell script on the front of a jar so the whole thing becomes self-contained (`mill` does this). I don't know if it's really any better. It confused me (and makes it a little difficult to package).